webcomponents/tool/src/main/frontend/js/tasks/sakai-tasks-create-task.js

```diff
@@ -226,13 +233,6 @@ export class SakaiTasksCreateTask extends SakaiDialogContent {
     this.siteId = this.siteIdBackup;
   }
 
-  complete(e) {
```

Comment:
I wanted to suggest an enhancement to our current approach. Currently, when the user clicks on the ```completed``` checkbox, it triggers the ```@click``` handler, which calls the ```complete()``` function as you mentioned.  However, I believe we can enhance the user experience by removing the ```complete()``` function and the ```@click``` handler.

Instead of maintaining a temporary save of the ```completed``` checkbox status, I propose sending the user inputs, including the status of the ```completed``` checkbox, to the server only when the user clicks on the ```save``` button. We can handle the ```completed``` checkbox just like other input fields such as ```description``` and ```priority```, ensuring consistency in the saving process. This approach eliminates the need for temporary storage of the ```completed``` checkbox status.

The reason behind this suggestion is that currently, if a user checks the ```completed``` checkbox and then closes the pop-up, the next time they click on the ```edit``` button, the ```completed``` checkbox remains checked. This may confuse the user because the task is not actually completed.

What do you think about it?